OracleDatabase/SingleInstance/dockerfiles/21.3.0/setupLinuxEnv.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2021 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: December, 2016
+# Author: gerald.venzl@oracle.com
+# Description: Sets up the unix environment for DB installation.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /opt/oracle as user 'oracle' to proceed with Oracle installation
+# ------------------------------------------------------------
+mkdir -p $ORACLE_BASE/scripts/setup && \
+mkdir $ORACLE_BASE/scripts/startup && \
+mkdir -p $ORACLE_BASE/scripts/extensions/setup && \
+mkdir $ORACLE_BASE/scripts/extensions/startup && \
+ln -s $ORACLE_BASE/scripts /docker-entrypoint-initdb.d && \
+mkdir $ORACLE_BASE/oradata && \
+mkdir -p $ORACLE_HOME && \
+chmod ug+x $ORACLE_BASE/*.sh && \
+yum -y install oracle-database-preinstall-19c openssl && \
```

Comment:
This should be the 21c preinstall rpm.

Reply:
At the time of first commit, oracle-database-preinstall-21c was not available for download, I updated it now and upgrade to use OL8

Reply:
Used oracle-database-preinstall-21c in commit 82b7492dd9615f56a7696bf04f26674ba907578d.